token/js/cli/token-test.js

```diff
@@ -417,18 +417,29 @@ export async function multisig(): Promise<void> {
 export async function nativeToken(): Promise<void> {
   const connection = await getConnection();
 
-  const mintPublicKey = new PublicKey(
-    'So11111111111111111111111111111111111111112',
-  );
+  const mintPublicKey = NATIVE_MINT;
```

Comment:
Would you mind just removing `mintPublicKey` entirely?  We can use `NATIVE_MINT` instead